sdk/search/search-documents/src/serviceUtils.ts

```diff
@@ -397,7 +397,7 @@ export function extractOperationOptions<T extends OperationOptions>(
   obj: T
 ): {
   operationOptions: OperationOptions;
-  restOptions: Pick<T, Exclude<keyof T, keyof OperationOptions>>;
+  restOptions: any;
```

Comment:
Rather than reduce the type safety here, it looks like this helper is only relevant to a single operation (SearchIndexClient.analyzeText) - can we instead remove this helper and do any needed conversion in that single operation?

Reply:
Done